posts/2021-03-03-jakarta-ee-9-compatibility.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Open Liberty beta is Jakarta EE 9 compatible
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jhanders34
+author_github: https://github.com/jhanders34
+seo-title: Open Liberty beta is Jakarta EE 9 compatible - OpenLiberty.io
+seo-description: Open Liberty beta releases have satisified the Jakarta EE 9 TCK and have been added to the Jakarta EE website.
+blog_description: "Open Liberty beta releases have satisified the Jakarta EE 9 TCK and have been added to the Jakarta EE website"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
```

Comment:
need to add a line fo the jakarta tag, per discussion from liberty-blogs channel and issue https://github.com/OpenLiberty/openliberty.io/issues/1934

tags: [JakartaEE]

Reply:
Added